### PR TITLE
try re-enabling http2 interop tests on helix

### PR DESCRIPTION
**Pull request by analogrelay**
The referenced issue is closed and even if it's still there we should try to resolve this and get the test in Helix. Opening a PR to try.

TODO:
* [x] Check for these results in Kusto

### CODE REVIEW

File: src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs [hunk 1]
```diff
@@ -1189,6 +1189,7 @@ public async Task Settings_HeaderTableSize_CanBeReduced_Server(string scheme)
         // Nor does Kestrel yet support sending dynamic table updates, so there's nothing to test here. https://github.com/dotnet/aspnetcore/issues/4715
 
         [Theory]
+        [QuarantinedTest]
```
Comment (anurse) [inline src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs]: I've also added this to https://github.com/dotnet/aspnetcore/pull/19568 , should merge OK.

**Comment by anurse** [comment]
Looks like this worked. Too bad @Tratcher is OOF now :). I don't see a reason why not to enable these if they're working though, we can always quarantine/skip them again. I'll double check Kusto to make sure the data is flowing and the tests are running.

**Comment by anurse** [comment]
Looks good. Here's a query counting Interop.FunctionalTests results from this PR (it includes multiple runs so don't judge the count much, just checking that it's non-zero).

  | WorkItemFriendlyName | count_
-- | -- | --
  | Interop.FunctionalTests--netcoreapp5.0 | 1436

**Review by jkotalik — APPROVED** [review]

**Comment by jkotalik** [comment]
Run it.

**Pull request merged**